src/__stories__/index.ts

```diff
@@ -1,2 +1,3 @@
 // NOTE: The ordering of these imports determines the ordering in Storybook
 import './JsonSchemaViewer';
+import './SchemaRow';
```

Comment:
```suggestion
```
we probably no longer need it do we?

Reply:
@jgreen44 was trying to review this but storybook will not start for me. Can you make a follow up PR and remove this?

<img width="783" alt="Screen Shot 2021-11-24 at 3 18 04 PM" src="https://user-images.githubusercontent.com/47359669/143314067-5cef7a89-f6e2-47cd-8661-755840b9a3f2.png">

Reply:
@paulatulis : I'm sorry, I'm not sure I understand what you mean by "make a follow up PR and remove this"

Reply:
It looks like you import SchemaRow into this file, but I believe it needs to be removed per @P0lip 's suggested change above. When you check out the main branch and try running `yarn storybook`, are you able to get it started? Maybe it's just me?

Reply:
Storybook has been removed. I will make another PR to remove `./SchemaRow` import

Reply:
(I believe she is referring to the line removal suggestion Jakub made)